components/Admin/index.test.tsx

```diff
@@ -85,7 +78,9 @@ describe('Admin page', () => {
           <Admin />
         </Tree>
       )
-      fireEvent.click(screen.getByLabelText('edit-signer'))
+      // not sure why this stopped working...
+      // fireEvent.click(screen.getByLabelText('edit-signer'))
```

Comment:
@navFooh not sure why i can't seem to getByLabelText here when it was working before. I looked over the previous code and nothing in the `aria-label` changed? I might be misunderstanding something here